inputfiles/addedTypes.json

```diff
@@ -3700,19 +3685,6 @@
                     }
                 }
             },
-            "HTMLVideoElement": {
```

Comment:
I don't see an existing entry where this HTMLVideoElement method is added. Was that an incorrect artifact of the merge too?

Reply:
I was confused at first, but looks like you've added it by hand after problematic merge.
https://github.com/microsoft/TSJS-lib-generator/pull/811#discussion_r393833305

Reply:
Here it is in file https://github.com/microsoft/TSJS-lib-generator/blob/af44ea1019180f60aced843d65a1fd048adb5735/inputfiles/addedTypes.json#L2265